app/assets/javascripts/widgets/wysiwyg.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  var anchorLinks = $('h3.anchorLinks a, a[href="#top"]');
+  var navHeight = $("#cu_nav").outerHeight();
+
+  /* offset omninav when interal anchor links scroll to target */
+  anchorLinks.on('click', function(e){
+    var target = $('[name=' + this.hash.slice(1) + ']');
```

Comment:
I want to believe there is a way to access the `target` var as a property of the `e` object rather than having to run a jQuery selector to get it.

But your explanation does make what's going on here a lot clearer. If you keep this, can you add that info to comment?